config-model/src/main/java/com/yahoo/vespa/model/container/search/ContainerSearch.java

```diff
@@ -113,6 +119,20 @@ public void getConfig(PageTemplatesConfig.Builder builder) {
         if (pageTemplates!=null) pageTemplates.getConfig(builder);
     }
 
+    private String buildGCOpts(Zone zone) {
+        Optional<String> gcopts = owningCluster.getGCOpts();
+        if (gcopts.isPresent()) {
+            return gcopts.get();
+        } else if (zone.system() == SystemName.dev) {
```

Comment:
Is this safe? Perhaps we should only enable G1 for dev in hosted until Vespa 7.

Reply:
No, we do not want to wait. For the applications there shall be no difference going from 6.x to 7. compared to 6.x to 6.x+1 unless you are using deprecated features. Then it will break.